Spigot-Server-Patches/0394-Asynchronous-chunk-IO-and-loading.patch

```diff
@@ -1,10 +1,11 @@
-From 867219c4f28a3b9b5ea9dfe010569d45b620f1e1 Mon Sep 17 00:00:00 2001
+From 2569817269d8854ce427a42e1880785f5ab7cf26 Mon Sep 17 00:00:00 2001
 From: Spottedleaf <Spottedleaf@users.noreply.github.com>
 Date: Sat, 13 Jul 2019 09:23:10 -0700
 Subject: [PATCH] Asynchronous chunk IO and loading
 
 THIS PATCH NEEDS RE-EVALUTING AND WILL LIKELY NOT WORK AS-IS RIGHT THIS SECOND
```

Comment:
Feel free to remove this warning, this is basically what I was planning to do, our IO stuff is nicer